src/NzbDrone.Core/Indexers/Definitions/Redacted.cs

```diff
@@ -1,23 +1,56 @@
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using FluentValidation;
 using NLog;
 using NzbDrone.Common.Http;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Annotations;
 using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Exceptions;
+using NzbDrone.Core.Indexers.Gazelle;
+using NzbDrone.Core.IndexerSearch.Definitions;
 using NzbDrone.Core.Messaging.Events;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
 
 namespace NzbDrone.Core.Indexers.Definitions
 {
-    public class Redacted : Gazelle.Gazelle
+    public class Redacted : TorrentIndexerBase<RedactedSettings>
     {
         public override string Name => "Redacted";
         public override string BaseUrl => "https://redacted.ch/";
+        public override string Description => "Redacted is a Private Torrent Tracker for Music";
+        public override string Language => "en-us";
+        public override Encoding Encoding => Encoding.UTF8;
+        public override DownloadProtocol Protocol => DownloadProtocol.Torrent;
         public override IndexerPrivacy Privacy => IndexerPrivacy.Private;
+        public override IndexerCapabilities Capabilities => SetCapabilities();
```

Comment:
We can also set SupportsRedirect to true for this given it builds standalone download links that don't require cookies to work

`public override bool SupportsRedirect => true;`